metisp/pymetis/src/pymetis/recipes/img/metis_lm_img_basic_reduce.py

```diff
@@ -59,7 +60,9 @@ class InputSet(PersistenceInputSetMixin, LinearityInputSetMixin, DarkImageProces
         # `RawImageProcessor.InputSet`. It already knows that it wants a RawInput and MasterDarkInput class,
         # but does not know about the tags yet. So here we define tags for the raw input:
         class RawInput(RawInput):
-            _tags = re.compile(r"LM_IMAGE_(?P<target>SCI|SKY|STD)_RAW")
+            _tags: re.Pattern = re.compile(r"LM_IMAGE_(?P<target>SCI|SKY|STD)_RAW")
+            # FIXME (or better, fix the DRLD): SKY is not documented, but it is requested by other recipes.
+            #    Martin put this there to generate all the data and thus pass tests. If it is not correct, shout at him.
```

Comment:
Wouldn't want anyone to shout at you, so created an issue in the DRLD repo
```suggestion
            #    See https://github.com/AstarVienna/METIS_DRLD/issues/321
```